src/com/facebook/buck/android/MergeAndroidResourcesStep.java

```diff
@@ -216,29 +230,47 @@ public StepExecutionResult execute(ExecutionContext context)
               filesystem,
               useOldStyleableFormat);
 
+      ImmutableSet.Builder<String> requiredPackages = ImmutableSet.<String>builder();
+
+      // Create a temporary list to avoid concurrent modification problems.
```

Comment:
Not sure this comment makes sense anymore.  Maybe comment why this has to be created before you reassign rDotJavaPackageToResources below?

Reply:
improved the comment